### ISSUE TITLE
Prevent duplicate epidnumbers via import [0.5]

### ISSUE THREAD

**Issue by MartinWahnschaffeSymeda** [hzi-braunschweig/SORMAS-Project#1393]
Those cases should not be imported and instead be added to the "error" report

**Comment by ChristopherRiedelSymeda** [comment]
Ticket will be postponed until it is exactly clarified how we want to deal with the non-existing cross-disease uniqueness.

**Comment by MartinWahnschaffeSymeda** [comment]
Do after #1404

**Comment by ChristopherRiedelSymeda** [comment]
Move new code up after the try block above

**Comment by MartinWahnschaffeSymeda** [comment]
Import does not work correctly now. The check for a duplicate epid number has to be done after the search for similar cases. This means when no duplicate is found or the user decides to pick, override or create.
You can reproduce this using CaseImporterTest.testImportAllCases.
